tests/unit_tests/test_validator.py

```diff
@@ -0,0 +1,322 @@
+import json
+from unittest.mock import mock_open, patch
+
+from goodmap.data_validator import (
+    FieldViolation,
+    FormatViolation,
+    ViolationType,
```

Comment:
Ok. This is kind of philosophic question... But are they part of interface?
Wouldn't just datapoint be enough to test it?

Reply:
`FieldViolation` and `FormatViolation` have different constructors. So I don't know how to get around this.